### PR TITLE
Update QueryRouteEnhancer::enhance to fix php 7.4 notices

### PR DESCRIPTION
**Pull request by indytechcook**
Update QueryRouteEnhancer::enhance to fix php 7.4 notices.  See #982

---
This change is [Reviewable](https://reviewable.io/reviews/drupal-graphql/graphql/989)

### CODE REVIEW

File: src/Routing/QueryRouteEnhancer.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ class QueryRouteEnhancer implements EnhancerInterface {
    */
   public function enhance(array $defaults, Request $request) {
     $route = $defaults[RouteObjectInterface::ROUTE_OBJECT];
-    if (!$route->hasDefault('_graphql')) {
+    if (!$this->applies($route)) {
```
Comment (indytechcook) [inline src/Routing/QueryRouteEnhancer.php:20]: I think I resolved that incorrectly.  I'll rebase.

Comment (hugovk) [reply src/Routing/QueryRouteEnhancer.php:20]: Please could you also fully enable PHP 7.4 on the CI, like [`6d767ee` (#1052)](https://github.com/drupal-graphql/graphql/pull/1052/commits/6d767ee54e2891402fad2b0bede9c303e966985a)? Thanks!

Comment (hugovk) [reply src/Routing/QueryRouteEnhancer.php:20]: We've used a patch based on this PR with just the `return` change below:

```php
return $defaults + ['operations' => $operations];
```

Would be great to get this PR fixed up and merged!

Thanks!

**Comment by bronius** [comment]
Patch no longer applies in 3.1-dev bc the new interface `EnhancerInterface` does not specify a method `::applies()` provided by deprecated `RouteEnhancerInterface` interface (https://www.drupal.org/node/2894934). Still, the deprecated method looks to be like the original:
```
  /**
   * {@inheritdoc}
   */
  public function applies(Route $route) {
    return $route->hasDefault('_graphql');
  }
```
Does `$route->hasDefault('_graphql');` cause php 7.4 notices?

**Comment by Luigisa** [comment]
@indytechcook @bronius 
I believe this request has already been resolved in the latest version 3.x update.
If you see it the same as me, could you please close this request?

**Comment by bronius** [comment]
> this request has already been resolved in the latest version 3.x update

Sounds reasonable. Thanks for looking in and grooming the open issues. Closing.

oh - I don't have a Close button, so I guess I lack permission to close it myself.

**Pull request closed without merging**